src/components/contact/sections/ContactUs.jsx

```diff
@@ -1,20 +1,22 @@
-import React, { useState } from "react";
+import React from "react";
 import { Container, Col, Form, Button, Nav } from "react-bootstrap";
-
+import emailjs from "emailjs-com";
 const ContactUs = () => {
-  const initContactFormState = {
-    firstName: "",
-    lastName: "",
-    email: "",
-    message: "",
-  };
-  const [form, setForm] = useState(initContactFormState);
-  const handleSubmit = (event) => {
-    event.preventDefault();
-    // TODO (basil-kawak) : connect this to contact us
-    alert(`you have submited youe message ${form}`);
-    setForm(initContactFormState);
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    emailjs
+      .sendForm(
+        "default_service",
+        "template_Uz0KFyDM",
+        e.target,
+        "user_31hCfU5FLOydTyY8iOnWx"
+      )
+      .then((result) => {
+        alert("Thank you, Your messeage have been sent");
```

Comment:
if you are planning to use this for the demo, please don't use an `alert`

Reply:
use setState or something and make it show a bootstrap box/alert or something like this